packages/sdk/src/contracts/factory.contract.ts

```diff
@@ -232,15 +250,9 @@ export class FactoryContract extends HolographBaseContract {
    * @param toChain The destination chain id.
    * @param sender The address of person making the request.
    * @param payload The payload of the request.
-   * @returns The function selector, and its calldata.
+   * @returns An array with the bridgeOut function selector and the provided payload
    */
-  async bridgeOut(
-    chainId: number,
-    toChain: number,
-    sender: Address,
-    payload: Hex,
-    wallet?: {account: string | HolographWallet},
```

Comment:
why did you remove the `wallet` param?

Reply:
Since this is a pure function in the contract, it doesn't alter the state and thus does not generate a transaction.
![image](https://github.com/holographxyz/holograph/assets/37748828/66d91fc4-a13e-4656-a79d-5b9aa921e9f5)